doc/getting_started/beginner_tutorial.rst

```diff
@@ -2,13 +2,10 @@ Beginner Tutorial
 =================
 Welcome to the beginner tutorial for Pachyderm. If you've already got Pachyderm installed, this guide should take about 15 minutes and you'll be introduced to the basic concepts of Pachyderm.
 
-Analyzing Log Lines from a Fruit Stand
---------------------------------------
+Image processing with OpenCV
+----------------------------
 
-In this guide you're going to create a Pachyderm pipeline to process
-transaction logs from a fruit stand. We'll use two standard unix tools, ``grep``
-and ``awk`` to do our processing. Thanks to Pachyderm's processing system we'll
-be able to run the pipeline in a distributed, streaming fashion. As new data is
+In this guide we're going to create a Pachyderm pipeline to do some simple edge detection on a few images. Thanks to Pachyderm's processing system, we'll be able to run the pipeline in a distributed, streaming fashion. As new data is
```

Comment:
Maybe link edge detection to https://en.wikipedia.org/wiki/Edge_detection.  Not guaranteed everyone will know right off the bat depending on background.